stabilizer/src/main/java/com/hazelcast/stabilizer/coordinator/Coordinator.java

```diff
@@ -372,6 +376,31 @@ private void echo(String msg) {
         log.info(msg);
     }
 
+    private void copyUploadDirectoryToAgents() throws IOException {
+        log.info("WORKING DIRECTORY:" + WORKING_DIRECTORY);
+        if (!UPLOAD_DIRECTORY.exists()) {
+            log.debug("Skipping upload, since no upload file in working directory");
+            return;
+        }
+        log.info("UPLOAD DIRECTORY:" + UPLOAD_HOME);
```

Comment:
Make the messages of the start/end log roughly the same so that the user knows that they are matching. So I would do something like: 

log.info(format("Starting uploading '+%s+' to agents", UPLOAD_HOME));